website/docs/r/pi_instance.html.markdown

```diff
@@ -64,6 +64,12 @@ Review the argument references that you can specify for your resource.
 - `pi_anti_affinity_instances` - (Optional, String) List of pvmInstances to base storage anti-affinity policy against; required if requesting `anti-affinity` and `pi_anti_affinity_volumes` is not provided.
 - `pi_anti_affinity_volumes`- (Optional, String) List of volumes to base storage anti-affinity policy against; required if requesting `anti-affinity` and `pi_anti_affinity_instances` is not provided.
 - `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+- `pi_deployment_target` - (Optional, List) The deployment of a dedicated host. MaxItems:1.
```

Comment:
Is Max Items here's the maximum length of the list? As in you can at most specify one target?
```suggestion
- `pi_deployment_target` - (Optional, List) The deployment of a dedicated host. Max Items: 1.
```